### ISSUE TITLE
Install license agreement

### ISSUE THREAD

**Issue by multiplemonomials** [Amber-MD/cmake-buildscripts#117]
CMake needs to install the `AmberTools/GNU_LGPL_V3` and `AmberTools/LICENSE` files to the root of the install prefix.

@dacase Which license should be shown in the actual installer?  Should I use a different one for Amber vs AmberTools?

**Comment by dacase** [comment]
The two license files cited above are in the AmberTools directory, and apply only for AmberTools.

There is a README file in the top-level directory that covers Amber.  But this README file should be installed for both the complete distribution and for the AmberTools-only installation.

**Comment by multiplemonomials** [comment]
OK, I started working on this but ran into a bit of an issue.  The top-level README has this text:

> Except as noted below, Amber 18 is provided under a license that has
restrictions on its use and redistribution.  You should not use Amber 18
unless you have executed a license agreement with UCSF.  Consult that
license to see the provisions that apply.

> The files in the "AmberTools" subdirectory are covered by a separate, open
source, license; see ./AmberTools/LICENSE for more information.

In the CMake-built Amber, there is no 'AmberTools' directory in binary distributions; components from AmberTools just get installed into `amber/bin` and `amber/lib`.  Clearly this clause won't apply any more.

How should this text in the license be reworded @dacase ?

**Comment by dacase** [comment]
This is a tough one.  The way we have always distinguished the licenses is by having an AmberTools directory.  Here's my current thinking (I'm open to arguments for change):

For binary distributions, we should only include AmberTools.  

The reasoning is that people wanting Amber (which is high-perforamnce part) have to sign a separate license, and get sent a pointer
to the source code.  This makes sense, since for pmemd, one might want to use Intel or PGI compilers for speed; for pmemd.MPI, you might need to link to a specialized, location-specific MPI library; for pmemd.cuda, one needs to compile with nvcc that is compatible with the CUDA sdk and drivers located on the particular machine.

Bottom line: Amber users are probably going to want to /need to compile from source anyway.
They can still use CMake, but they don't need to have a binary version.

Then, for the AmberTools binary distribution, put LICENSE and GNU_LGPL_V3 in the top-level directory.

**Comment by multiplemonomials** [comment]
Yeah that makes sense I guess.  I always assumed the the utility of Amber binary distributions would be for sysadmins in an organization to distribute and install Amber across their computers.  So, it it's only going to be distributed privately, does that mean we shouldn't install any license files at all when building full Amber?

An alternative idea I had is to just put the following clause at the top of CMake's license file:
```
* The Amber programs, including all variants of the "pmemd" executable, are
  under the Amber proprietary license available here: 
  http://ambermd.org/LicenseAmber18.pdf
```
I feel like that should cover it since the PMEMD executable is really the only part of Amber included in binary distributions.  

Another issue with the licensing is that the main license file refers to several other license files for subprograms, with paths that will become invalid if it is installed in the top level directory.  Currently, I have CMake set up to create a `amber/licenses` top-level directory, and install all subsidiary license files in there.   This also requires creating a slightly different version of the AmberTools license file with the paths changed to e.g. `licenses/reduce/LICENSE`.  Does that sound OK to you?

**Comment by dacase** [comment]
I think we should stick for now with only having a binary distribution for AmberTools.  There are thorny issues (some discussed above) with how to handle the licensing for the full distribution, and additional problems figuring out how to distribute the latter.

(Note that for source code, we have two distros, AmberTools and Amber, and the latter requires the former, but does not include it.  If we are to go to a binary distro for Amber, I'd think we would follow the same model.  Of course, that doesn't really solve the problem of where to put the license files.)

**Comment by multiplemonomials** [comment]
So, does that mean we should actually block users from building binary distributions that contain PMEMD?
